src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -26,6 +26,10 @@
      ? strrchr(__FILE__,'/')+1                  \
      : __FILE__                                 \
 )
+
+/* TODO: make it a configure option */
+#define MPIDI_OFI_MAX_CONTEXTS                  16
```

Comment:
Since we use `vni` or `vci` as the variable name for VCI "index", I find naming the actual structure also "VNI" confusing. At the meantime, I like the existing code that simply call it `ctx[]`, so strictly speaking, I am not bring it back but rather keeping the existing. Semantically, I find context quite fit to what we really mean here. But certainly the naming is not set-in-stone at this point.

Oh, if you are just commenting on this particular macro, I am fine calling it `MPIDI_OFI_MAX_VNI` -- in fact, I'll change it.

Reply:
Well, I was referring to any names related to VNI (or endpoint, context, whatever...) in OFI netmod, not particularly for this macro.

Names in OFI netmod were defined far before we invent the term VCI/VNI at CH4 level, so just because we have the name "context" in OFI netmod doesn't mean we should keep using it forever.

But... given the fact that OFI already uses the terms "endpoint" and "context" in its manpage, it makes some sense to keep using these terms.

Maybe all I want is a clear comment saying that the abstract construct "VNI" is mapped to "OFI context" in case of OFI netmod.